tests/integration/plugins/test_plugins.py

```diff
@@ -93,11 +93,18 @@ async def _wait_for_units(ops_test: OpsTest, deployment_type: str) -> None:
             apps=[APP_NAME],
             apps_statuses=["active"],
             units_statuses=["active"],
-            wait_for_exact_units={APP_NAME: 3},
```

Comment:
why is this deleted?

Reply:
I originally had one single `wait_until` for both COS and opensearch and was unsure how unit is counted with subordinates. Let me readd it as we broke the waits now.